tutorials/notebooks/Coordinates-Intro/Coordinates-Intro.ipynb

```diff
@@ -45,6 +45,7 @@
     "from astropy import units as u\n",
     "from astropy.coordinates import SkyCoord\n",
     "from astropy.table import Table\n",
```

Comment:
The changes in this file seem unrelated to this PR.

Reply:
I was trying to get Circle CI and Travis CI passed, since some errors related to this file was thrown after I fixed the broken logo link.

Reply:
No problem, but let's fix the travis build in a separate PR, since this is a change to the tutorial *content*